packages/git/src/browser/style/history.css

```diff
@@ -113,4 +131,50 @@
       bottom: -80px  ;
     }
   }
-  
\ No newline at end of file
+
+.theia-git .git-diff-container .subject {
+    font-size: var(--theia-ui-font-size2);
+    font-weight: bold;
+}
+
+.theia-git .git-diff-container .commit-info {
+    padding-left: 10px;
+    box-sizing: border-box;
+    overflow: hidden;
+}
+
+.theia-git .git-diff-container .commit-info-row {
+    align-items: center;
+    margin-top: 10px;
+}
+
+.theia-git .commit-info .header-row {
+    margin: 4px 0;
+}
+
+.theia-git .commit-info .header-row .theia-header {
+    margin: 1px 0;
+}
+
+.theia-git .commit-info .header-row .header-value {
+    margin: 0 0 0 5px;
+}
+
+.theia-git .commit-info-row .image-container {
+    display: flex;
+}
+
+.icon-git-commit {
+    width: 20px!important;
```

Comment:
we should have a separate css class for width, height, it is relevant only in the history context, it is better to move generic octicons css classes to another file